### ISSUE TITLE
Integration with 🤗 Hub

### ISSUE THREAD

**Issue by sayakpaul** [Deci-AI/super-gradients#577]
Hi folks. 

Thanks for providing the pre-trained models along with pre-training scripts. 

At Hugging Face, the Hub is our house to serve models, datasets, spaces, etc. It facilitates easy artifact loading and usage, providing common and streamlined API access to your models, datasets, etc. 

Hugging Face supports third-party integrations too and I was wondering if you'd be up to exploring the integration. The integration will primarily facilitate easy model sharing and model downloading which could be beneficial for the vision community in general. 

Here's the main doc that'd be helpful for you for the integration: https://huggingface.co/docs/hub/models-adding-libraries. Let me know if you'd need any help.
